test/standalone/test-debuglet.js

```diff
@@ -145,7 +145,17 @@ describe(__filename, function(){
     debuglet.start();
   });
 
-  it('should error if a package.json doesn\'t exist');
+  it('should error if a package.json doesn\'t exist', function(done) {
+    var config = extend({}, defaultConfig, {projectId: 'fake-project', workingDirectory: __dirname});
+    debuglet = new Debuglet(fakeDebug, config, logger);
+
+    debuglet.once('initError', function(err) {
+      assert(err);
```

Comment:
nit: prefer `assert.ifError(err)`  for readability.. The rest of the file has counter examples, but it would be good to prefer `assert` methods when possible.